server/game/cards/events/06/savingthekingdom.js

```diff
@@ -0,0 +1,40 @@
+const DrawCard = require('../../../drawcard.js');
+
+class SavingTheKingdom extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Make character unable to stand',
+            phase: 'challenge',
+            target: {
+                activePromptTitle: 'Select a character',
+                cardCondition: card => card.location === 'play area' && card.getType() === 'character' && card.getStrength() <= 3
+            },
+            handler: context => {
+                this.untilEndOfRound(ability => ({
```

Comment:
The card text specifies that it kneels the card as well. So you'll need to kneel the card, plus add `gameAction: 'kneel'` to the target properties.